### ISSUE TITLE
Stabalization - roadmap to v1.0

### ISSUE THREAD

**Issue by tcharding** [rust-bitcoin/rust-secp256k1#662]
Tracking issue for what is needed for us to be able to stabalize the `secp256k1` crate.

### Issues we want resolved

- 

### Issues that might apply

-

**Comment by tcharding** [comment]
Originally discussed in https://github.com/rust-bitcoin/rust-bitcoin/discussions/1392

**Comment by tcharding** [comment]
@apoelstra please add a `1.0` label same as `rust-bitcoin`.

name: 1.0
color: #78BE0C
description: Issues and PRs required or helping to stabilize the API

**Comment by apoelstra** [comment]
Done.

**Comment by tcharding** [comment]
Thanks man

**Comment by Kixunil** [comment]
FTR, I think this crate is **not** blocking stabilization of important `bitcoin` types since we could hide `secp256k1::*` types and operations and only expose them in serialized form or via extension traits. As such I suggest we focus most of our time at stabilizing relevant parts of `bitcoin`.

**Comment by tcharding** [comment]
Yep sure thing, @apoelstra also told me offline that he didn't want to focus on secp at the moment. Mainly I was wanting to keeping on eye on secp so we don't do anything that we later have to change because of secp and for things that create/prove conventions (like error stuff etc.). I'll try a bit harder to stay away from here though :)

**Comment by apoelstra** [comment]
FWIW "at the moment" just means "since I just had a baby". I don't mean to imply that I'm stepping away from this project or that I plan for it to be indefinitely stalled.

**Comment by tcharding** [comment]
My bad, that was my understanding, i didn't mean to misrepresent you.
